plugins/woocommerce/tests/e2e-pw/tests/merchant/product-reviews.spec.js

```diff
@@ -1,63 +1,142 @@
+					type: 'simple',
+					regular_price: '9.99',
+				} )
+				.then( ( response ) => {
+					product = response.data;
+				} );
+
+			// Create the product review
+			await api
+				.post( 'products/reviews', {
+					product_id: product.id,
+					review: `Nice one, Playwright! ${ timestamp }`,
+					reviewer: 'John Doe',
+					reviewer_email: `john.doe.${ timestamp }@example.com`,
+					rating: ( Math.random() * ( 5 - 1 ) + 1 ).toFixed( 0 ),
+				} )
+				.then( ( response ) => {
+					review = response.data;
+				} );
+
+			await use( { product, review } );
+
+			// Cleanup
+			await api.delete( `products/reviews/${ review.id }`, {
+				force: true,
+			} );
+			await api.delete( `products/${ product.id }`, {
+				force: true,
+			} );
+		},
+	} );
+
+	test( 'can view product review', async ( { page, testData } ) => {
+		await page.goto(
+			`wp-admin/edit.php?post_type=product&page=product-reviews`
+		);
+		await expect(
+			page.getByRole( 'cell', { name: testData.review.review } )
+		).toBeVisible();
+		await expect(
+			page.getByRole( 'link', { name: testData.product.name } )
+		).toBeVisible();
+		await expect(
+			page.getByLabel( `${ testData.review.rating } out of 5` )
+		).toBeVisible();
+		await expect(
+			page.getByRole( 'link', { name: testData.review.reviewer_email } )
+		).toBeVisible();
 	} );
 
-	test.afterAll( async ( {} ) => {
-		await api.delete( `products/${ productId }`, {
-			force: true,
-		} );
+	test( 'can edit product review', async ( { page, testData } ) => {
+		await page.goto(
+			`wp-admin/edit.php?post_type=product&page=product-reviews`
+		);
+		await expect(
+			page.getByRole( 'cell', { name: testData.review.review } )
+		).toBeVisible();
+
+		//Hover over the product review to have 'Edit' displayed
+		await page.hover( '.comment-text' );
+
+		// Create new review, Quick Edit it and save
+		const EditedtimeStamp = Date.now().toString();
+		const updatedReview = `(edited ${ EditedtimeStamp })`;
+		await page.getByRole( 'button', { name: 'Quick Edit' } ).click();
+		await page.locator( '.wp-editor-area' ).first().fill( updatedReview );
+		await page.getByRole( 'button', { name: 'Update Comment' } ).click();
+
+		// Verify that the edited comment is there
+		const commentText = page.locator( '.comment-text' );
+		await commentText.waitFor( { state: 'visible' } );
+		const updatedComment = await commentText.innerText();
+		await expect( updatedComment ).toBe( updatedReview );
 	} );
 
-	test( 'can view product review', async ( { page } ) => {
+	test( 'can delete product review', async ( { page, testData } ) => {
 		await page.goto(
 			`wp-admin/edit.php?post_type=product&page=product-reviews`
 		);
 		await expect(
-			page.getByRole( 'cell', { name: productReview } )
+			page.getByRole( 'cell', { name: testData.review.review } )
 		).toBeVisible();
+		const reviewerName = testData.review.reviewer;
+
+		//Hover over the product review to have 'Edit' displayed
+		await page.hover( '.comment-text' );
```

Comment:
Similar to the `edit` test, need an unique locators so it doesn't fail if there are more reviews in the table.

```suggestion
		await page.hover( `#comment-${ testData.review.id }` );
```